README.md

```diff
@@ -125,7 +134,9 @@ This section will walk you through the steps to get started with pgai and Ollama
     
 1. **Search the embeddings**
 
-    We'll search the embeddings for the concept of "properties of light"
+    We'll search the embeddings for the concept of "properties of light" even though these words are not in the text of the articles. This is only possible because of vector embeddings that capture the semantic meaning of the text.
```

Comment:
```suggestion
    We'll search the embeddings for the concept of "properties of light" even though these words are not in the text of the articles. This is possible because vector embeddings capture the semantic meaning of the text.
```